.github/ISSUE_TEMPLATE/Question.md

```diff
@@ -0,0 +1,40 @@
+---
+me: Question 
+about: Any support questions you might have.
+
+---
+
+## Question
```

Comment:
It might be nice to understand the context of the question. 

something like `Are you asking about community best practices, how to implement a specific feature, or about general context and help around the operator-sdk`? 

Thoughts?

Reply:
Yeah, that could make it easier for us when we're quickly looking through recent issues.

Reply:
@shawn-hurley where were you thinking of putting the above text? As a comment or as a section?

Reply:
I was thinking it would be a section similar to the `Operator-SDK Version:` above. I think we may want to put this before the Question header?